packages/fiori-elements-writer/test/unit/defaults.test.ts

```diff
@@ -40,11 +40,11 @@ describe('Defaults', () => {
               "autoHide": undefined,
               "entityConfig": Object {
                 "mainEntityName": "",
+                "tableType": "AnalyticalTable",
               },
               "multiSelect": undefined,
               "qualifier": undefined,
```

Comment:
Strange that `qualifier` is still specified under the template setting root rather than the entity config (as per type, although Im not sure thats correct either)

Reply:
Good catch. Thanks.